PythonVisualizations/orderedArray.py

```diff
@@ -0,0 +1,491 @@
+        self.list[j]= (drawable(
+            val, self.canvas.itemconfigure(cellPair[0], 'fill'), *cellPair))
+
+<<<<<<< Updated upstream
+        self.window.update()  
+        self.cleanUp(callEnviron)
+        
+    def remove(self, val):
+        callEnviron = self.createCallEnvironment()
+        self.startAnimations()
+        index = self.find(val)
+        found = index != None    # Record if value was found
+        if found:
+            self.wait(0.3)
+=======
+        self.window.update()       
+        
+    def remove(self,val):
+        self.startAnimations()
+        index = self.find(val)
+        if index != None:
+            self.wait(1)
+            self.cleanUp()
+>>>>>>> Stashed changes
+
+            n = self.list[index]
+
+            # Slide value rectangle up and off screen
+            items = (n.display_shape, n.display_val)
+            self.moveItemsOffCanvas(items, N, sleepTime=0.02)
+
+            # Create an index for shifting the cells
+            kIndex = self.createIndex(index, 'k')
+            callEnviron |= set(kIndex)
+            
+            # Slide values from right to left to fill gap
+            for i in range(index+1, len(self.list)):
+                self.assignElement(i, i-1, callEnviron)
+                self.moveItemsBy(kIndex, (CELL_SIZE, 0), sleepTime=0.01)
+
+            self.removeFromEnd()
+<<<<<<< Updated upstream
+        
+        self.cleanUp(callEnviron)
+        return found
+=======
+            self.stopAnimations()
+            return True
+        self.stopAnimations()
+        return False 
+>>>>>>> Stashed changes
+    
+    def randomFill(self):
+        # Clear the list so new values can be entered
+        self.list=[] 
+        size = self.size
+        
+        # Create a list of random numbers and sort them
+        a = [random.randrange(90) for i in range(size)]
+        a.sort()
+        
+        # Append and draw them to the list and draw them
+        for i in a:
+            self.list.append(drawable(i))
+        
+        self.display()         
+        
+    def newArraySize(self, val):
+        # Clear Array and reset size and list
+        self.canvas.delete("all")
+        self.size = val
+        self.list = []        
+        
+        for i in range(val):  # Draw new grid of cells
+            self.createArrayCell(i)        
+        
+        self.window.update()
+
+    def removeFromEnd(self):
+        callEnviron = self.createCallEnvironment()
+        self.startAnimations()
+        # pop a Drawable from the list
+        if len(self.list) == 0:
+            self.setMessage('Array is empty!')
+            return
+        n = self.list.pop()
+
+        # delete the associated display objects
+        self.canvas.delete(n.display_shape)
+        self.canvas.delete(n.display_val)
+        
+<<<<<<< Updated upstream
+=======
+        print(len(self.list))
+>>>>>>> Stashed changes
```

Comment:
Another merge conflict.  The print statement is probably for debugging purposes and should be removed in the version that is committed.